pkg/analyzer/analyzers/square/permissions.yaml

```diff
@@ -0,0 +1,44 @@
+permissions:
+  - BANK_ACCOUNTS_READ
+  - APPOINTMENTS_WRITE
+  - APPOINTMENTS_ALL_WRITE
+  - APPOINTMENTS_READ
+  - APPOINTMENTS_ALL_READ
+  - APPOINTMENTS_BUSINESS_SETTINGS_READ
```

Comment:
Is there a reason why these are all capitalized? I don't recall any other analyzer doing that.

Reply:
I think I copied it from somewhere. You are right, all of our analyzers have permissions in small caps. Let me change this one in the name of consistency.

Reply:
@mcastorina updated the permissions with small caps for consistency.